ar-setup.sh

```diff
@@ -121,7 +121,7 @@ add_gitlab_project_vars() {
 	gitlab_project=${gitlab_project//\//%2F}
 
 	# check whether the GitLab project exists
-	output_project_existence=$(curl -s --header "PRIVATE-TOKEN: ${gitlab_access_token}" "${gitlab_hostname}api/v4/projects/${gitlab_project}")
+	output_project_existence="$(curl -s --header "PRIVATE-TOKEN: ${gitlab_access_token}" "${gitlab_hostname}api/v4/projects/${gitlab_project}")"
```

Comment:
nested quotes. You need to escape the nested ones.

Reply:
Check https://google.github.io/styleguide/shellguide.html#quoting:
```
# "quote command substitutions"
# Note that quotes nested inside "$()" don't need escaping.
flag="$(some_command and its args "$@" 'quoted separately')"
```

Reply:
Oh, okay. Good to know no escaping is needed. Please ignore my other comments.